src/components/editions/byline.tsx

```diff
@@ -160,14 +165,14 @@ const bylineSecondaryStyles = (
 `;
 
 const getStyles = (format: Format, kickerColor: string): SerializedStyles => {
-	if (format.design === Design.Interview) {
-		return css(styles(kickerColor), interviewStyles);
-	}
-
 	if (format.display === Display.Immersive) {
 		return css(styles(kickerColor), immersiveStyles);
 	}
 
+	if (format.design === Design.Interview) {
+		return css(styles(kickerColor), interviewStyles);
+	}
+
```

Comment:
Since the ordering matters, maybe add a comment here and in the other place you're reordering in case someone in the future wants to rearrange?

Reply:
Good idea 👍